pkgs/development/ocaml-modules/dns/default.nix

```
@@ -16,15 +16,15 @@
   ohex,
 }:
 
-buildDunePackage rec {
+buildDunePackage (finalAttls: {
```

```suggestion
buildDunePackage (finalAttrs: {
```

typed too fast maybe 🤣

and 2 more below